setup.py

```diff
@@ -71,17 +71,13 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        # 'marisa_trie',
-        # 'networkx',
-        # 'rdflib',
-        'nltk',
-        'numpy',
-        'scipy',
-        # 'pandas',
-        'gensim',
-        'sklearn',
-        # 'Click',
-        'keras',
+        'nltk==3.2.2',    # most of these can probably be upgraded later
+        'numpy==1.12.0',  # but keras should stay at 1.1.1
+        'scipy==0.18.1',
+        'gensim==0.13.4.1',
+        'scikit-learn==0.18.1',
+        'keras==1.1.1',
+        'h5py==2.6.0',
```

Comment:
@jstypka what about pinning the major version, so that we can at least upgrade to bugfixes?
E.g.:
```
nltk~=3.2
numpy~=1.12
scipy~=0.18
...
```